docs/Examples/generate-blocks-and-txs.js

```diff
@@ -0,0 +1,172 @@
+'use strict';
```

Comment:
This script is useful for testing purposes, however is starting a new node and chain with different consensus rules and is in memory only. This can be done via RPC commands, including setting mock time, and I think more useful in the automated and manual e2e test cases.

Reply:
I am also having an issue running the script, it is giving:
```
Creating wallets and accounts...
Mining initial blocks...
Air-dropping funds to the people...
Confirming airdrop...
Creating a big mess!...
Problem sending tx: Error: Not enough funds. (available=0.0, required=0.0009506)
Problem sending tx: Error: Not enough funds. (available=0.00115604, required=0.00151922)
Problem sending tx: Error: Not enough funds. (available=0.00084805, required=0.00099687)
Problem sending tx: Error: Not enough funds. (available=0.00018887, required=0.00137865)
Problem sending tx: Error: Not enough funds. (available=0.00018887, required=0.00176489)
Problem sending tx: Error: Not enough funds. (available=0.00040918, required=0.0005302)
Problem sending tx: Error: Not enough funds. (available=0.0008495, required=0.00116315)
All done! Go play.
```

Reply:
The `not enough funds` is expected. Transactions are generated randomly, so even though a few don't go through, there are still dozens of transactions. Is it worth fixing this script? Or are you thinking I should remove it? I added it here because there have been questions about calls like `estimatesmartfee` which require several blocks full of transactions to return anything meaningful. So the idea is to run this script, then experiment with `fee` calls by interacting with the in-memory node.

Reply:
I think it's useful to demonstrate as it's pretty common with application development testing.